models/friendship.js

```diff
@@ -1,22 +1,26 @@
-const Sequelize = require('sequelize');
-
-const db = require('../../models/index');
-
-const friends = 'friends';
-
-
-const Friendship = db.sequelize.define('Friendship', {
-    userOneId: {
-      type: Sequelize.INTEGER,
-    },
-    userTwoId: {
-      type: Sequelize.INTEGER,
-    },
-    status: {
-        type: Sequelize.TINYINT,
-      }
-
-  }, { friends });
-
-
-module.exports = Friendship; 
+'use strict';
```

Comment:
Here, which one of model is that what we need? My model is already on master.

Reply:
There are 2 models 1 is auto-genereted and built into db and second is for validating upcoming data to the API, that's why we have 2 models directories

Reply:
I have to check if those auto-generated ones can be used for the API related so we will be able to stick with one of them